codalab/worker/upload_util.py

```diff
@@ -61,16 +67,33 @@ def upload_with_chunked_encoding(
 
```

Comment:
Is this the exact same fn as the one in upload_manager.py? If so we might want to refactor / unify in a future PR

Reply:
Essentially. There are a few minor differences. I agree that it should be refactored.